config/tsconfig.base.json

```diff
@@ -0,0 +1,31 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
+    "moduleResolution": "node",
+    "strict": true,
+    "noImplicitAny": true,
+    // "noImplicitThis": true,
+    "strictNullChecks": true,
+    "strictFunctionTypes": true,
+    "strictBindCallApply": true,
+    "strictPropertyInitialization": true,
+    // "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    // "noImplicitReturns": true,
+    "noFallthroughCasesInSwitch": true,
+    "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true,
+    "skipLibCheck": true,
+    "forceConsistentCasingInFileNames": true,
+    "resolveJsonModule": true,
+    // "isolatedModules": true,
+    "noEmit": true, // ensure that nobody can accidentally use this config for a build
+    "sourceMap": true,
+    "removeComments": true,
+    "downlevelIteration": true,
+    // "paths": {
+    //   "@project-leek/*": ["./packages/*/src"] // TODO check if helpful
```

Comment:
Is this TODO still necessary ?

Reply:
I will have a closer look to it, but wanted to get this PR done for now, so I left it as a comment in the file.